.github/workflows/check-evangelists-links.yml

```diff
@@ -0,0 +1,24 @@
+on:
+  schedule:
+    - cron:  '0 2 1 * *'
+
+name: Broken Link Check
+jobs:
+  check:
+    name: Broken Link Check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 13.x
+      - name: Install Dependencies
+        run: |
+          npm install -g mkdirp
```

Comment:
Not needed?

Reply:
For some reason it was failing without it, ex: https://github.com/kingthorin/wstgjob/runs/635326509?check_suite_focus=true

Reply:
Looks like an issue, that's not declared as a dependency.

Reply:
I'll look into reporting it, thanks!

Reply:
Thank you :)

Reply:
Reported: https://github.com/timaschew/link-checker/issues/10